spec/support/features/session_helper.rb

```diff
@@ -261,7 +261,7 @@ def sign_in_live_with_piv_cac(user = user_with_piv_cac)
 
     def fill_in_code_with_last_phone_otp
       accept_rules_of_use_and_continue_if_displayed
-      fill_in :code, with: last_phone_otp
+      fill_in I18n.t('forms.two_factor.code'), with: last_phone_otp
```

Comment:
this helper is usually called in a non-JS browser, so it was fine

However the locale picker requires JS, and running the tests with JS now scrambles the ID for the OTP field (#5594) so this was updated to use the label instead

Reply:
>this was updated to use the label instead

👍  Generally I'd be in favor of this as a default approach, since users interact with fields by labels, not by markup IDs, so this makes the tests more closely resemble how the app is used.